gp2gp-translator/src/integrationTest/java/uk/nhs/adaptors/pss/util/JsonPathIgnoreGeneratorUtil.java

```diff
@@ -68,6 +74,24 @@ private static boolean isList(Resource resource, String listDisplay) {
         return false;
     }
 
+    private static Boolean isUnknownPractitioner(IBaseResource resource) {
+        Practitioner practitioner = (Practitioner) resource;
+
+        return UNKNOWN_USER.equals(practitioner.getNameFirstRep().getText());
+    }
+
+    private static Boolean isUnknownPerformer(Resource resource) {
+        ProcedureRequest procedureRequest = (ProcedureRequest) resource;
+
+        IBaseResource performer = procedureRequest.getPerformer().getResource();
+
+        if (performer instanceof Practitioner) {
+            return isUnknownPractitioner(performer);
+        }
+
+        return false;
```

Comment:
You could use ternary operator:
```suggestion
        return performer instanceof Practitioner ? isUnknownPractitioner(performer) : false;
```